kjdoc/tour.md

```diff
@@ -1021,6 +1021,19 @@ KJ provides an advanced, cross-platform filesystem API in `kj/filesystem.h`. Fea
 * Symlinks, hard links, listing directories, recursive delete, recursive create parents, recursive copy directory, memory mapping, and unnamed temporary files are all exposed and easy to use.
 * Sparse files ("hole punching"), copy-on-write file cloning (`FICLONE`, `FICLONERANGE`), `sendfile()`-based copying, `renameat2()` atomic replacements, and more will automatically be used when available.
 
+```c++
+auto fs = kj::newDiskFilesystem();
```

Comment:
This needs a comment clarifying that you are recommended to construct this at the outermost part of your program (i.e. main) & dependency inject the implementation where needed (that way, for example, you could have a test implementation that roots you somewhere or be given a user space network filesystem implementation)

Reply:
Sure, done.